aten/src/ATen/native/cuda/LinearAlgebra.cu

```diff
@@ -1,7 +1,7 @@
-#define TORCH_ASSERT_NO_OPERATORS
```

Comment:
Why is this removed?

Reply:
Because we added the dependency with `BatchLinearAlgebra.h` down there.

This one was tricky, as we don't have a file to have our custom kernels for linalg. This is, by the looks of it, the only linalg function where we use a TensorIterator. It's no big deal, as this file is super short, so removing that macro from here is fine really.